HIP/hip-765.md

```diff
@@ -10,41 +10,16 @@ status: Accepted
 last-call-date-time: 2023-07-28T07:00:00Z
 created: 2023-07-11
 discussions-to: https://github.com/hashgraph/hedera-improvement-proposal/pull/765
-updated: 2023-08-01
+updated: 2024-01-23
 ---
 
 ## Abstract
 
-
-This HIP proposes the addition of the metadata field to Non-Fungible Token Class, taking after the individual Non-Fungible Token (NFT) metadata field which was added in HIP-17. While HIP-17 metadata is scoped to a single NFT, here we propose adding metadata scoped to an entire token class.
-
-## Motivation
-
-Similar to NFT's on the network, the fungible token implementation is largely functional. There is a need for a token creator to specify additional "metadata" about the token. The data and formatting is explored in HIP-YYY but there is no field suitable for storing the link to this metadata. 
-
-The most 'logical' field proposed to store this data is the existing 'memo' field, however a dedicated metadata field is more appropriate.
-
-## Rationale
-
-The success of HIP-412 in standardizing the individual NFT space on Hedera demonstrates the potential of standardized metadata functionality for NFT Collections as well. Wallets, explorers and other applications can display relevant supplementary information about tokens to users in a decentralized fashion.
-
-## User stories
-
-As a token creator, I want a platform-standardized method to host information about my NFT Collection in a decentralized fashion, allowing applications to query data such as token icon, featured image and website.
-
-As a wallet or explorer, I want to query a mirror node to obtain the metadata for any NFT Collection token, similar to current individual non-fungible tokens. Mirror nodes currently return metadata for individual NFTs; but for an NFT Collection token genrally, we want to get **Token-level** (ie Collection-level) metadata.
-
-As a developer, I want to store the metadata uri in a dedicated field, to avoid ambiguity of using an overloaded field such as 'memo'.
-  
-## Specification
-
-## HAPI Changes
-=======
 This HIP proposes the addition of the metadata field to Non-Fungible Token Class, taking after the individual Non-Fungible Token (NFT) metadata field, which was added in [HIP-17](https://hips.hedera.com/hip/hip-17). While HIP-17 metadata is scoped to a single NFT, here we propose adding metadata scoped to an entire token class.
```

Comment:
We should list the related HIPs 646 and 657 and note that 646 is adding the same fields.

Reply:
Done